cli/tests/testdata/task/deno_json/task_no_args.out

```diff
@@ -1,19 +1,19 @@
 Available tasks:
 - boolean_logic
     sleep 0.1 && echo 3 && echo 4 & echo 1 && echo 2 || echo NOPE
-- deno_echo
-    deno eval 'console.log(5)'
 - echo
     echo 1
+- deno_echo
+    deno eval 'console.log(5)'
+- strings
+    deno run main.ts && deno eval "console.log(\"test\")"
+- piped
+    echo 12345 | (deno eval 'const b = new Uint8Array(1);Deno.stdin.readSync(b);console.log(b)' && deno eval 'const b = new Uint8Array(1);Deno.stdin.readSync(b);console.log(b)')
+- exit_code_5
+    echo $(echo 10 ; exit 2) && exit 5
```

Comment:
Not super happy about this change - alphabetical ordering in the output seems desirable

Reply:
It was by mistake that I had made it alphabetical in the beginning. The reason for this is so that it shows up exactly as it is in the file. Many people group similar commands together for example.

Reply:
Discussed offline, it seems people like to logically group tasks so retaining the original order seems desirable